ncl/ncl_entries/great_circle.ipynb

```diff
@@ -180,6 +188,44 @@
     ")"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## gc_qarea\n",
+    "NCL's `gc_qarea` calculates the area of a (four-sided) quadrilateral patch on the unit sphere\n",
+    "\n",
+    "<div class=\"admonition alert alert-info\">\n",
+    "    <p class=\"admonition-title\" style=\"font-weight:bold\">Important Note</p>\n",
+    "    Coordinates should be within the valid latitude/longitude range (-90° to 90° and -180° to 180°) and be in clockwise or counter-clockwise order\n",
+    "</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Grab and Go"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pyproj import Geod\n",
+    "\n",
+    "# Points in clockise order: Roughly Four Corners of Colorado\n",
+    "latitudes = [41.00488, 41.00203, 37.00540, 37.00051]  # degrees\n",
+    "longitudes = [-109.05001, -102.05348, -103.04633, -109.04720]  # degrees\n",
+    "\n",
+    "geod = Geod(ellps=\"sphere\")  # radius = 6370997 m\n",
```

Comment:
Should we at least demo here how to set the radius and/or do the calculation on the unit sphere as in `gc_qarea` (i.e. with `a=1`)?

I realize it's not a big jump to figure that out on your own, but given that original function produced results the unit sphere and it's not hard to demo I think it makes sense.

It also shows more clearly how you might set a different radius, which could be of interest for folks working with model data where the earth is approximated as a sphere and the radius is set to something slightly custom (not an uncommon case it turns out).

Open to being convinced otherwise.